diff -urN libfm-0.1.17/configure.ac libfm-0.1.17-patched/configure.ac
--- libfm-0.1.17/configure.ac	2011-10-24 22:30:05.000000000 +0200
+++ libfm-0.1.17-patched/configure.ac	2012-05-30 05:16:21.000000000 +0200
@@ -3,12 +3,15 @@
 
 AC_PREREQ([2.63])
 AC_INIT([libfm], [0.1.17], [http://pcmanfm.sorceforge.net/])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([-Wall])
 AC_CONFIG_MACRO_DIR(m4)
 AC_CONFIG_HEADERS([config.h])
 
 # Checks for programs.
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
 AC_PROG_CC
+AM_PROG_CC_C_O
 AM_PROG_LIBTOOL
 
 # Checks for libraries.
