--- libfm-1.2.0/Makefile.am~	2013-12-05 23:05:58.000000000 +0800
+++ libfm-1.2.0/Makefile.am	2014-02-20 08:49:50.971894574 +0800
@@ -37,11 +37,15 @@
 endif
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = \
-	libfm.pc \
-	libfm-gtk.pc \
-	libfm-gtk3.pc \
-	$(NULL)
+pkgconfig_DATA = libfm.pc
+
+if USE_GTK2
+pkgconfig_DATA += libfm-gtk.pc
+endif
+
+if USE_GTK3
+pkgconfig_DATA += libfm-gtk3.pc
+endif
 endif
 
 DISTCHECK_CONFIGURE_FLAGS=--enable-gtk-doc
--- libfm-1.2.0/configure.ac~	2014-02-20 09:03:34.615196776 +0800
+++ libfm-1.2.0/configure.ac	2014-02-20 09:06:48.480682351 +0800
@@ -144,6 +140,8 @@
 AC_SUBST(LIBFM_GTK_LTLIBRARIES)
 AC_SUBST(LIBFM_PREF_APPS)
 AM_CONDITIONAL([ENABLE_GTK], [test x$ac_with_gtk != xnone])
+AM_CONDITIONAL([USE_GTK3], [test x$ac_with_gtk = x3.0])
+AM_CONDITIONAL([USE_GTK2], [test x$ac_with_gtk = x2.0])
 
 dnl pango_modules="pango >= 1.16.0"
 dnl PKG_CHECK_MODULES(PANGO, [$pango_modules])
